### DISCUSSION TITLE
WPS305 Found `f`-string

### DISCUSSION THREAD

**Discussion by cidrblock** [ansible/ansible-navigator#717] [Linters, linting, formatting, pre-commit, CI] [answered]
https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.FormattedStringViolation

this conflicts with pylint C0209

```
Formatting a regular string which could be a f-string Used when we detect a string that is being formatted with format() or % which could potentially be a f-string. The use of f-strings is preferred. Requires Python 3.6 and py-version >= 3.6.
```

**Comment by webknjaz** [comment, marked as answer]
I usually disable this one. It makes sense for codebases that support Pythons below 3.6 but not for us.
